logic/logic_test.go

```diff
@@ -160,3 +161,24 @@ func TestCheckProgramV4(t *testing.T) {
 	err = CheckProgram(program, args)
 	require.NoError(t, err)
 }
+
+func TestCheckProgramV5(t *testing.T) {
```

Comment:
I think there are a few other opcodes in V5 like cover, uncover, log, etc. May be good to add tests for those, too.

Reply:
Good idea